Jellyfin.Server/Program.cs

```diff
@@ -101,7 +118,7 @@ private static async Task StartApp(StartupOptions options)
             SQLitePCL.Batteries_V2.Init();
 
             // Allow all https requests
-            ServicePointManager.ServerCertificateValidationCallback = new RemoteCertificateValidationCallback(delegate { return true; });
+            ServicePointManager.ServerCertificateValidationCallback = new RemoteCertificateValidationCallback(delegate { return true; } );
```

Comment:
On an unrelated note, this should be configurable. I believe this is for out going connections, and it's not very good practice to have it as default.

Reply:
But probably also out of scope. It might warrant a `//TODO`

Reply:
Yeah, this is a bad practice, should be off by default.
Definitely warrants a TODO at least.